core/src/main/java/io/kestra/core/runners/pebble/filters/DistinctFilter.java

```diff
@@ -0,0 +1,36 @@
+package io.kestra.core.runners.pebble.filters;
+
+import io.pebbletemplates.pebble.error.PebbleException;
+import io.pebbletemplates.pebble.extension.Filter;
+import io.pebbletemplates.pebble.template.EvaluationContext;
+import io.pebbletemplates.pebble.template.PebbleTemplate;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class DistinctFilter implements Filter {
+
+    @Override
+    public List<String> getArgumentNames() {
+        return null;
+    }
+
+    @Override
+    public Object apply(Object input, Map<String, Object> args, PebbleTemplate self, EvaluationContext context,
+                        int lineNumber) throws PebbleException, NullPointerException {
```

Comment:
```suggestion
                        int lineNumber) throws PebbleException {
```

Please don't declare throws NullPointerException, NPE are always considered as a bug.